child_compassion/security/sponsorship_groups.xml

```diff
@@ -11,5 +11,10 @@
             <field name="comment">the user will have access to all sponsorships and related data.</field>
             <field name="category_id" ref="module_category_compassion"/>
         </record>
+        <record id="group_admin" model="res.groups">

```

Comment:
I didn't see you added this group. You can remove it and use `base.erp_manager` instead.